src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -938,37 +938,40 @@ users:
 +
 Example:
 ....
-java -jar /opt/proactive/node.jar
+-Dproactive.communication.protocol=pamr -Dproactive.useIPaddress=true -Dproactive.pamr.router.address=192.168.99.1
 ....
 
- - *jvmProperties:* Java options appended to the command used to start the node on the remote host. As the <<../admin/ProActiveAdminGuide.html#_proactive_message_routing,ProActive Message Routing>> is required by the Kubernetes Infrastructure, PAMR properties must be defined.
+- *startupScript:* VM startup script to launch the ProActive nodes. To run the ProActive nodes, this script is also expected to download and install the Java Runtime Environment if it is not already installed in the specified image, then download and and execute ProActive node.jar. The script can use <<_cloud_infrastructure_startup_script,some arguments>>. For example, it can use `%nodeJarUrl%` to represent the full URL path for downloading the _node.jar_ file. These arguments will be interpreted by Kubernetes Infrastructure later. This is an optional property. If left blank, a default script is automatically generated for the Linux OS.
 +
-Example:
+Default startupScript:
 ....
--Dproactive.communication.protocol=pamr -Dproactive.useIPaddress=true -Dproactive.pamr.router.address=192.168.99.1
+mkdir -p /tmp/node && cd /tmp/node
+if ! command -v wget; then apt-get update; apt-get -y install wget; fi
+wget -nv --no-check-certificate %nodeJarUrl%
+if ! command -v java; then wget -nv -N https://s3.amazonaws.com/ci-materials/Latest_jre/jre-8u131-linux-x64.tar.gz; tar -xf jre-8u131-linux-x64.tar.gz; export PATH=/tmp/node/jre1.8.0_131/bin/:$PATH; fi
+java -jar node.jar -Dproactive.communication.protocol=%protocol% -Dpython.path=%jythonPath% -Dproactive.pamr.router.address=%rmHostname% -D%instanceIdNodeProperty%=%instanceId% -r %rmUrl% -s %nodeSourceName% %nodeNamingOption% -v %credentials% -w %numberOfNodesPerInstance% %additionalProperties%
 ....
 
- - *kubernetesNamespace:* (Optional) Kubernetes namespace if different from _default_ namespace.
- - *image:* Docker image used to deploy ProActive Nodes. This image must contain a Java Runtime Environment. Example: java:8-jre
- - *nodeTimeout:* (Optional) The estimated startup time of the nodes (expressed in millisecond). After this timeout expired, the node will be considered as lost. Default value is 10 minutes.
- - *cleaningDelaySeconds:* (Optional) Cleaning delay, in seconds until cleaning service starts. Default value is 120 seconds.
- - *cleaningFrequencySeconds:* (Optional) Cleaning frequency, Kubernetes node cleaning is done every x seconds. Default value is 30 seconds.
+- *nodeTimeout:* (Optional) The estimated startup time of the nodes (expressed in millisecond). After this timeout expired, the node will be considered as lost. Default value is 10 minutes.
```

Comment:
After this timeout expired => After this timeout. Because timeout expresses already expiration :))